keyboards/dztech/dz65rgb/keymaps/pagondel/keymap.c

```diff
@@ -0,0 +1,47 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
```

Comment:
We recommend enums over defines, since they're easier to maintain long term. 
```suggestion
enum layers {
    _BL,
    _FL,
};
```

Reply:
replaced defines with enums as requested